src/CommunityToolkit.Maui.Markup/AppBuilderExtensions.cs

```diff
@@ -5,13 +5,35 @@
 /// </summary>
 public static class AppBuilderExtensions
 {
+	static IServiceProvider serviceProvider =>
+#if WINDOWS
+		MauiWinUIApplication.Current.Services;
+#elif ANDROID
+		MauiApplication.Current.Services;
+#elif IOS || MACCATALYST
+		MauiUIApplicationDelegate.Current.Services;
+#elif TIZEN
+		throw new NotImplementedException();
```

Comment:
Hey @JoonghyunCho! Could you implement this line of code? It just needs to reference Tizen's `IServiceProvider` implementation.

Reply:
Is there a more elegant way to get IServiceProvider? This code may exist in .NET MAUI. In that case we do not need to add additional target frameworks

Reply:
I hope so. I'd really prefer to keep `<TargetFramework>net7.0</TargetFramework>` as our only TFM.